src/pages/SearchPage.js

```diff
@@ -169,7 +169,10 @@ class SearchPage extends Component {
         );
 
         return (
-            <ScreenWrapper includeSafeAreaPaddingBottom={false}>
+            <ScreenWrapper
+                includeSafeAreaPaddingBottom={false}
+                shouldDisableAutoFocus
```

Comment:
I disable the autofocus of the focus trap because in the `SearchPage` we have an element with autofocus (the input bar). If I didn't disable it, it would overwrite the autofocus of the input bar (video).

<details>
<summary>
Video with autofocus enabled
</summary>

https://github.com/Expensify/App/assets/104823336/41f019c6-8ebb-4df5-9bcf-95d836f06634

</details>

The `shouldDisableAutoFocus` disable the autofocus of the FocusTrap library. Normally, when the focus trap is activated, the library has to focus on some element inside the trap.

Reply:
You are right. It will be much safer to enable it case-by-case. In that way, we will avoid unnecessary regressions. I'm going to refactor the PR.